modules/imap/functions.php

```diff
@@ -1412,6 +1412,56 @@ function snooze_dropdown($output, $unsnooze = false) {
     return $txt;
 }}
 
+/**
+ * @subpackage imap/functions
+ */
+if (!hm_exists('get_forward_content')) {
+    function get_forward_content($format, $only_label = false) {
+        if ($format == 'Forward as message attachment') {
+            $label = 'Forward as message attachment';
+        } elseif ($format == 'Edit as new message') {
+            $label = 'Edit as new message';
+        }
+        if ($only_label) {
+            return [$label];
+        }
+    }}
+
+/**
+ * @subpackage imap/functions
+ */
+if (!hm_exists('forward_content')) {
```

Comment:
Same think here. No need to create a function.

Reply:
I just removed this function please review